<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserInfoChangerClass</class>
 <widget class="QWidget" name="UserInfoChangerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>205</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UserInfoChanger</string>
  </property>
  <widget class="QLabel" name="IntroLable">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>271</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ark Pixel 12px monospaced zh_cn</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>在此修改你的信息（无需修改的留空即可）🐦</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>271</width>
     <height>108</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>新用户名</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NamelineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>原密码   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="PastlineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>新密码   </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="NowlineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>再次输入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RelineEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="CommitpushButton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ClearpushButton">
      <property name="text">
       <string>清空</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
